public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -266,14 +273,18 @@ export const Explorer = ({
 
   const getErrorHandler = (title: string) => {
     return (error: any) => {
-      const formattedError = formatError(error.name, error.message, error.body.message);
-      notifications.toasts.addError(formattedError, {
-        title,
-      });
+      // const formattedError = formatError(error.name, error.message, error.body.message);
```

Comment:
Were the error toasts here commented out because of it being included in the query assistant summary? If so, should we delete these lines?